C/reversenumber.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+int main() {
+    int n, rev = 0, remainder;
```

Comment:
- use a better variable name instead of n
- add some comments to explain the logic of reversing the number